obal/data/modules/koji_find_tags.py

```diff
@@ -0,0 +1,30 @@
+"""
+Find all defined Koji tags in inventory
+"""
+
+from ansible.module_utils.basic import AnsibleModule
+
+def main():
+    """
+    Verify a tag
+    """
+    module = AnsibleModule(
+        argument_spec=dict(
+            packages=dict(type='dict', required=True)
+        )
+    )
+
+    packages = module.params['packages']
+
+    tags = set()
+
+    for attributes in packages.values():
+        if 'koji_tags' in attributes:
+            for tag in attributes['koji_tags']:
+                tags.add(tag['name'])
```

Comment:
What's the benefit of looping here instead of bulk adding a set?

Reply:
I suppose if I put a generator in there to extract the `name` field I could do a bulk add?

Reply:
Ah right, I missed it's extracting the name from `tag` and somehow assumed it was just adding `tag`. In that case, you can use this:
```python
tags += {tag['name'] for tag in attributes['koji_tags']}
```

Though I'm not sure which version states the intent more clearly.

@evgeni any opinion?

Reply:
```
TypeError: unsupported operand type(s) for +=: 'set' and 'set'
```

:-)

`tags.update({…})` would work tho.

I'm fine with either form.